src/daft-core/src/array/ops/list_agg.rs

```diff
@@ -10,52 +10,72 @@ use crate::{
     series::IntoSeries,
 };
 
+macro_rules! impl_daft_list_agg {
+    ($name:ty) => {
+        type Output = DaftResult<ListArray>;
+
+        fn list(&self) -> Self::Output {
+            let child_series = self.clone().into_series();
+            let offsets =
+                arrow2::offset::OffsetsBuffer::try_from(vec![0, child_series.len() as i64])?;
+            let list_field = self.field.to_list_field()?;
+            Ok(ListArray::new(list_field, child_series, offsets, None))
```

Comment:
I'm confused. why is validity always empty (`None`) in this case? is this intentional? At least comment why.

Reply:
There's no validity buffer here since everything is aggregated into one non-null row